Sources/DiagnosticsLogger.swift

```diff
@@ -143,6 +143,9 @@ extension DiagnosticsLogger {
                 return assertionFailure()
         }
 
+        // Make sure we have more then 500MB space left
+        guard UIDevice.current.freeDiskSpaceInBytes > (500 * 1024) else { return }
```

Comment:
Just to make sure I understand it correctly. We check for the device disk space, because we can't catch the out of disk space exception in Swift? Also, I might be wrong, but don't we check here if there is more than 500kb space left instead of 500 MB?

Reply:
Yes, I'll update that!